demisto_sdk/commands/common/tests/update_id_set_test.py

```diff
@@ -645,6 +645,26 @@ def test_get_script_data_with_alternative_fields_second_level():
 
         assert IsEqualFunctions.is_dicts_equal(returned_data, const_data)
 
+    @staticmethod
+    def test_get_script_data_script_executions_for_js_code():
+        """
+        Given
+            - A script file which contains javascript code.
+
+        When
+            - parsing scripts files
+
+        Then
+            - make sure the script_executions was parsed successfully.
+        """
+        file_path = TESTS_DIR + '/test_files/Script-js-script-code.yml'
+        script_name = 'TestScriptWithJsCode'
```

Comment:
Use the `TestSuite` object instead of creating a test data file.
see the recent tests of `process_layoutscontainer`